File: src/main/java/me/miki/shindo/injection/mixin/interfaces/client/renderer/IMixinRenderManager.java
package me.miki.shindo.injection.mixin.interfaces.client.renderer;

import net.minecraft.client.renderer.entity.RenderPlayer;

public interface IMixinRenderManager {
    double getRenderPosX();

    double getRenderPosY();

    double getRenderPosZ();

    RenderPlayer getPlayerRenderer();
}

File: src/main/kotlin/me/miki/shindo/injection/mixin/interfaces/client/renderer/IMixinRenderManager.kt
package me.miki.shindo.injection.mixin.interfaces.client.renderer

/**
 * Interface mixin para RenderManager. Usa Any para tipos do Minecraft para evitar LinkageError.
 */
interface IMixinRenderManager {
    fun getRenderPosX(): Double
    fun getRenderPosY(): Double
    fun getRenderPosZ(): Double
    fun getPlayerRenderer(): Any
}
